producers/mimosa26_sbg/src/M26Producer.cxx

```diff
@@ -0,0 +1,511 @@
+//#include "M26Controller.hh"
+#include "eudaq/Producer.hh"
+#include "eudaq/Logger.hh"
+#include "eudaq/RawDataEvent.hh"
+#include "eudaq/Timer.hh"
+#include "eudaq/Utils.hh"
+#include "eudaq/OptionParser.hh"
+#include "eudaq/Configuration.hh"
+
+#include "M26Producer.hh"
+
+// IRC exceptions
+#include "irc_exceptions.hh"
+
+//#include <stdio.h>
+//#include <stdlib.h>
+//#include <memory>
+#include <pthread.h>
+#include <sched.h>
+
+using eudaq::RawDataEvent;
+
+
+M26Producer::M26Producer(const std::string &name,
+    const std::string &runcontrol,
+    const std::string &verbosity)
+  : eudaq::Producer(name, runcontrol), m_producerName(name), m_terminated(false), m_configured(false), 
+  m_running(false), m_triggering(false), m_stopping(false) {
+    m_sumEnDetectors = 0;

```

Comment:
why this variable isn't in the initializer list but separately in the actual execution block?